<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmPwd</class>
 <widget class="QDialog" name="frmPwd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>密码修改</string>
  </property>
  <property name="windowIcon">
   <iconset resource="main.qrc">
    <normaloff>:/image/main.png</normaloff>:/image/main.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;宋体&quot;;</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>148</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="labPwdOld">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>原 密 码:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPwdOld">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labPwdNew">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>新 密 码:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPwdNew">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labPwdNewX">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>确认密码:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPwdNewX">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOk">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>确定(&amp;O)</string>
    </property>
    <property name="icon">
     <iconset resource="main.qrc">
      <normaloff>:/image/update.png</normaloff>:/image/update.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>关闭(&amp;C)</string>
    </property>
    <property name="icon">
     <iconset resource="main.qrc">
      <normaloff>:/image/delete.png</normaloff>:/image/delete.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtPwdOld</tabstop>
  <tabstop>txtPwdNew</tabstop>
  <tabstop>txtPwdNewX</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
